tracing/src/main/java/org/axonframework/extensions/tracing/MapExtractor.java

```diff
@@ -9,6 +24,10 @@
 import java.util.Iterator;
 import java.util.Map;
 
+/**
```

Comment:
Some javadoc describing what this (utility?) class does would be beneficial.

Reply:
Yep